packages/account/src/Components/trading-assessment/risk-tolerance-warning-modal.jsx

```diff
@@ -1,8 +1,31 @@
 import React from 'react';
-import { Button, Icon, Modal, Text } from '@deriv/components';
+import { Button, Icon, MobileDialog, MobileWrapper, Modal, Text } from '@deriv/components';
 import { Localize, localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
 
 const RiskToleranceWarningModal = ({ show_risk_modal, setShowRiskModal, title }) => {
+    if (isMobile()) {
+        return (
+            <MobileWrapper>
```

Comment:
I think when using MobileWrapper, u need not check isMobile()

Reply:
resolved 👍🏼